### PR TITLE
CSVS-105 Integration tests for account/vm existence check were implemented

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by MedvedevBW** [comment]
[run tests]

File: src/it/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceIntegrationTestSuite.scala [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      "test@example.com",
+      "firstname",
+      "lastName",
+      "password",
+      s"username $accountId"
+    )).withId(accountId).withName(accountName).withDomain(domainId)
+
+    accountDao.create(accountCreateRequest)
+
+    Thread.sleep(1000)
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceIntegrationTestSuite.scala]: add a note what we're waiting for

File: src/it/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceIntegrationTestSuite.scala [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    assert(cloudStackService.doesAccountExist(accountId))
+
+    val vmCreateTestRequest = new VmCreateTestRequest(VmCreateRequest.Settings(
+      retrievedServiceOfferingId, retrievedTemplateId, retrievedZoneId
+    )).withDomainAccount(accountName, domainId).asInstanceOf[VmCreateTestRequest]
+
+    val vmId = mapper.deserialize[VmCreateResponse](executor.executeRequest(vmCreateTestRequest.request)).vmId.id
+
+    Thread.sleep(3000)
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceIntegrationTestSuite.scala]: add a note what we're waiting for

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceIntegrationTestSuite.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceIntegrationTestSuite.scala]: ok

File: src/it/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceIntegrationTestSuite.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceIntegrationTestSuite.scala]: ok

**Review by placccebo — APPROVED** [review]

**Comment by MedvedevBW** [comment]
[run tests]

**Pull request merged**